ichnaea/scripts/importer.py

```diff
@@ -44,8 +43,9 @@ def load_file(session, source_file, batch_size=1000, userid=None):
                     # convert from unixtime to utc
                     time = datetime.datetime.utcfromtimestamp(time)
 
-                key = normalized_wifi_key(str(fields[1]))
-                if not valid_wifi_pattern(key):  # pragma: no cover
+                try:
+                    key = WifiKeyNode(String()).deserialize(str(fields[1]))
```

Comment:
I'm not a big fan of having to explicitly instantiate this validator like this (needing to import String from colander).  We should be able to use it by itself without needing to import additional things from colander.  I'm not sure what the best way around this is.